src/chttpd/src/chttpd.erl

```diff
@@ -16,6 +16,7 @@
 
 -include_lib("couch/include/couch_db.hrl").
 -include_lib("chttpd/include/chttpd.hrl").
+-include_lib("kernel/include/logger.hrl").
```

Comment:
Since we include `logger.hrl` in `chttpd.hrl`, should also include it here?

Reply:
Ah, I had gone back and forth on whether to include it in the header file(s) and just forgot to remove it. This is one topic I wanted to get feedback on. I ended up adding logger.hrl explicitly everywhere, but would it be better to embed in other headers? I don't have a strong preference.

Reply:
Ah makes sense. Having it everywhere is better it seems -- it's more explicit

Reply:
Removed nested header inclusion in 654baa7